hs_core/tests/api/rest/test_resource_list.py

```diff
@@ -263,3 +263,11 @@ def test_resource_list_by_bounding_box(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         content = json.loads(response.content)
         self.assertEqual(content['count'], 3)
+
+        # Bad coverage has no effect
+        response = self.client.get('/hsapi/resource/', {'coverage_type': 'bad',
+                                                        'nonsensical': '90',
+                                                        'params': '140'}, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
I am wondering if we change the serializer like this we may not have to change anything else. The api then should return 400 for the bad test case.
```coverage_type = serializers.CharField(required=False) ``` -> ```coverage_type = serializers.ChoiceField(choices=['box', 'point'], required=False)```